Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: help install install-dev format lint test test-cov check build run-container build-and-push deploy clean-deployment-stage
+.PHONY: help install install-dev format lint test test-cov check build run-container build-and-push deploy clean-deployment-stage bump-deps requirements
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)